src/pybind/mgr/dashboard/frontend/.stylelintrc

```diff
@@ -0,0 +1,36 @@
+{
+  "extends": "stylelint-config-sass-guidelines",
+  "plugins": [
+    "stylelint-declaration-use-variable"
+  ],  
+  "rules": {
+    "declaration-property-value-blacklist":null,
+    "function-parentheses-space-inside":null,
+    "indentation":null,
```

Comment:
`function-parentheses-space-inside` and `indentation` cause conflict with `npm run fix:prettier` and hence disabled,